Packs/MicrosoftTeams/ReleaseNotes/1_5_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Microsoft Teams
+
+- %%UPDATE_RN%%
```

Comment:
Please finish filling out the release notes in `Packs/MicrosoftTeams/ReleaseNotes/1_5_0.md`.
See https://xsoar.pan.dev/docs/documentation/release-notes.

Reply:
@samuelFain , please discard this PR. I've submitted a new one using the latest version of the integration, and included another fix where text is not wrapped properly when using MicrosoftAskUser automation. This is the new PR: 35592